pom.xml

```diff
@@ -61,6 +61,7 @@
         <joda.version>1.6.2</joda.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <confluent.maven.repo>http://packages.confluent.io/maven/</confluent.maven.repo>
+        <jackson.version>2.5.4</jackson.version>
```

Comment:
Any specific reason to use 2.5.4. I see that Kafka depends on 2.6.3 - can we use that version instead ?